language/python/develop.md

```diff
@@ -141,7 +141,7 @@ if __name__ == "__main__":
     app.run(host ='0.0.0.0')
 ```
 
-We’ve added the MySQL module and updated the code to connect to the database server, create a database and table. We also created a route to fetch widgets. We now need to rebuild our image so it contains our changes.
+We’ve added the MySQL module and updated the code to connect to the database server, created a database and table. We also created a route to fetch widgets. We now need to rebuild our image so it contains our changes.
```

Comment:
I believe it should be "create", as in "we've updated the code to connect and create".

"Created" makes it sound like we did two things, we've updated the code and we've created a database and table.

But we did only one thing, we've updated the code. We've updated the code so that it can connect to the database and create a database and table.

Reply:
@usha-mandya